content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -278,24 +278,16 @@ Make sure that your network manifest supports RBAC.
 Also, beware, that your Pod network must not overlap with any of the host networks as this can cause issues.
 If you find a collision between your network plugin’s preferred Pod network and some of your host networks, you should think of a suitable CIDR replacement and use that during `kubeadm init` with `--pod-network-cidr` and as a replacement in your network plugin’s YAML.
 
-You can install a pod network add-on with the following command on the control-plane node or a node that has the kubeconfig credentials:
+You can install a Pod network add-on with the following command on the control-plane node or a node that has the kubeconfig credentials:
 
 ```bash
 kubectl apply -f <add-on.yaml>
 ```
 
-You can install only one pod network per cluster.
+You can install only one Pod network per cluster.
+Below you can find installation instructions for some popular, CNCF approved Pod network plugins.
```

Comment:
@neolit123 I wonder about “CNCF approved”. Is there a list elsewhere of which CNI plugins have CNCF approval?

Reply:
the list i'm referring to is the CNCF "Cloud Native Network" landscape:
https://landscape.cncf.io/category=cloud-native-network&format=card-mode&grouping=category

but technically this sentence does not include the rest of the criteria we have, and we may or may not want to include it here...

should i just change this to the following?

> Below you can find installation instructions for some popular Pod network plugins:

Reply:
> Below you can find installation instructions for some popular Pod network plugins:

:+1: I would, it's &hellip; less controversial.